packages/suite/src/views/wallet/send/components/Outputs/index.tsx

```diff
@@ -28,8 +27,16 @@ const StyledCard = styled(Card)`
     gap: ${spacingsPx.xs};
 `;
 
-const StyledEvmExplanationBox = styled(EvmExplanationBox)`
-    margin-bottom: ${spacingsPx.md};
+const StyledEvmExplanation = styled.div`
+    background-color: ${({ theme }) => theme.backgroundAlertBlueSubtleOnElevation0};
+    color: ${({ theme }) => theme.textAlertBlue};
+    ${typography.hint}
```

Comment:
I think these css props shouldn't be necessary. 

Also you don't need override `CoinLogo` if you use `gap`: 

```js
const StyledEvmExplanation = styled.div`
  flex-direction: row;
  align-items: center;
  gap: ${spacingsPx.sm};
```

Reply:
Fixed in 13198aa4f845e30644728c57d9fb997eb7350066